ui/app/styles/core/helpers.scss

```diff
@@ -81,6 +81,7 @@
 .is-flex-start {
   display: flex !important;
   justify-content: flex-start;
+  gap: $spacing-m;
```

Comment:
spacing between buttons. confirmed only impacts the desired row of buttons.

Reply:
I wonder if this would be better in it's own utility class in case `is-flex-start` needs to be used without that gap spacing?

Reply:
Good call. Amended.